opentracing/__init__.py

```diff
@@ -58,3 +59,14 @@ def set_global_tracer(value):
 
     global tracer
     tracer = value
+
+    is_registered = True
```

Comment:
You need to declare `global is_registered` for it being actually set (as done with `global tracer`). Also, I'd rename it to `is_tracer_registered` ;)